logging/cluster-logging-support.adoc

```diff
@@ -29,3 +29,30 @@ The {logging-title} is not:
 * Historical or long term log retention or storage
 * A guaranteed log sink
 * Secure storage - audit logs are not stored by default
+
+include::modules/cluster-logging-maintenance-support-list.adoc[leveloffset=+1]
+include::modules/unmanaged-operators.adoc[leveloffset=+1]
+
+[id="cluster-logging-support-must-gather"]
+== Collecting logging data for Red Hat Support
+
+When opening a support case, it is helpful to provide debugging information about your cluster to Red Hat Support.
+
+The
+ifdef::openshift-enterprise,openshift-origin[]
+xref:../support/gathering-cluster-data.adoc#gathering-cluster-data[`must-gather` tool]
+endif::[]
+ifdef::openshift-rosa,openshift-dedicated[]
+link:https://docs.openshift.com/container-platform/latest/support/gathering-cluster-data.html#gathering-cluster-data[`must-gather` tool]
+endif::[]
+enables you to collect diagnostic information for project-level resources, cluster-level resources, and each of the {logging} components.
```

Comment:
```suggestion
You can use the
ifdef::openshift-enterprise,openshift-origin[]
xref:../support/gathering-cluster-data.adoc#gathering-cluster-data[`must-gather` tool]
endif::[]
ifdef::openshift-rosa,openshift-dedicated[]
link:https://docs.openshift.com/container-platform/latest/support/gathering-cluster-data.html#gathering-cluster-data[`must-gather` tool]
endif::[]
to collect diagnostic information for project-level resources, cluster-level resources, and each of the {logging} components.
```
Per [the ISG](https://www.ibm.com/docs/en/ibm-style?topic=grammar-anthropomorphism), "Do not assign human characteristics to inanimate objects, which are incapable of human behaviors and emotions. As much as possible, focus technical information on users and their actions, not on a product and its actions." Rephrase the sentence to make it based on user agency, rather than the technology "enabling" the user to do something. One suggested edit is above.